policy_set_integration_test.go

```diff
@@ -245,6 +266,8 @@ func TestPolicySetsRead(t *testing.T) {
 	orgTest, orgTestCleanup := createOrganization(t, client)
 	defer orgTestCleanup()
 
+	upgradeOrganizationSubscription(t, client, orgTest)
```

Comment:
Using the `upgradeOrganizationSubscription` in all policy set test. Prior to adding this, the policy set was all failing with `resource not found` on my dev.